en/includes/guides/authentication/mfa/add-x509-login.md

```diff
@@ -275,119 +307,83 @@ If {{product_name}} is already started, use the [Certificate Validation Manageme
+2.  If you want to store X.509 certificates as user claims, add the following property to the same file:
 
     ``` toml
     [authentication.authenticator.x509_certificate.parameters]
     setClaimURI = "http://wso2.org/claims/userCertificate"
     ```
 
-4.  To enable storing the X509 certificate as a user claim, add the following parameter.
+4.  To enable automatic self-registration for users who authenticate with an X.509 certificate, add the following property to the same file:
 
     ``` toml 
     [authentication.authenticator.x509_certificate.parameters]
     EnforceSelfRegistration = true
     ```
 
-5. Restart the server to apply the changes.
+5. Restart {{product_name}} to apply the changes.
 
-## Add a claim mapping for the certificate
+## Step 6: Add an attribute mapping for the certificate
 
-If storing the certificate as a user claim is enabled, the X509 certificate will be stored as a user claim and verified with the
-retrieved certificate from the request.
+If you have enabled storing X.509 certificates as a user attribute, {{product_name}} will save the certificate as a user attribute and validate it against the certificate presented in the authentication request.
 
-To add the custom attribute, follow the [Add custom attributes](../../../guides/users/attributes/manage-attributes.md/#add-custom-attributes) steps 
-and use the following details for the claim addition.
+To add the custom attribute, follow the steps in [add custom attributes]({{base_path}}/guides/users/attributes/manage-attributes.md/#add-custom-attributes) and use the following details.
 
-``` text
 - Attribute name : userCertificate
 - Attribute Display Name : User Certificate
-```
-![add-user-certificate-attribute](../../../assets/img/guides/authentication/mfa/add-user-certificate-attribute.png){: width="600" style="display: block; border: 0.3px solid lightgrey;"}
-
-This will create the **OpenID Connect** and **SCIM 2.0** protocol mappings as well. When storing the certificate in a user attribute, you will 
-need to update the column size of the `VALUE` column of the `UM_USER_ATTRIBUTES` table to a suitable value.
 
-## Import certificate
+![add-user-certificate-attribute]({{base_path}}/assets/img/guides/authentication/mfa/add-user-certificate-attribute.png){: width="600" style="display: block; border: 0.3px solid lightgrey;"}
 
--   **Chrome**
-    1.  In your browser, go to **Settings** > **Privacy and security** > **Manage certificates** > **Your certificates**.
-    
-        ![manage-cert-chrome](../../../assets/img/guides/authentication/mfa/manage-certificates-chrome.png){: width="800" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+This will also generate the **OpenID Connect** and **SCIM 2.0** protocol mappings. If you store the certificate as a user attribute, you must adjust the column size of the VALUE column in the UM_USER_ATTRIBUTES table to accommodate the certificate data.
```

Comment:
```suggestion
This will also generate the **OpenID Connect** and **SCIM 2.0** protocol mappings. If you store the certificate as a user attribute, you must adjust the column size of the `VALUE` column in the `UM_USER_ATTRIBUTES` table to accommodate the certificate data.
```